test/test_port.cpp

```diff
@@ -30,155 +30,227 @@ class PortUnitTest : public PnetUnitTest
 
 TEST_F (PortUnitTest, PortGetPortList)
 {
-   pnet_t * dummystack = NULL;
+   pnet_t dummystack;
    pf_lldp_port_list_t port_list;
 
    memset (&port_list, 0xff, sizeof (port_list));
+   memset (&dummystack, 0, sizeof (dummystack));
```

Comment:
Can we move boilerplate setup code to the text fixture class?

Reply:
Good point.
Moving to `class PortUnitTest`
Done